ray-operator/main.go

```diff
@@ -83,12 +84,13 @@ func main() {
 		"Forced cluster upgrade flag")
 	flag.StringVar(&logFile, "log-file-path", "",
 		"Synchronize logs to local file")
+	flag.StringVar(&logFileEncoder, "log-file-encoder", "json",
+		"Encoder to use for log file. Valid values are 'json' and 'console'. Defaults to 'json'")
 	flag.BoolVar(&ray.EnableBatchScheduler, "enable-batch-scheduler", false,
 		"Enable batch scheduler. Currently is volcano, which supports gang scheduler policy.")
 	flag.StringVar(&configFile, "config", "", "Path to structured config file. Flags are ignored if config file is set.")
 
 	opts := k8szap.Options{
-		Development: true,
```

Comment:
What is this for and why do we decide to remove it?

Reply:
Development mode has some defaults that are undesirable. Specifically the encoder defaults to `console` and debug level is logged: https://github.com/kubernetes-sigs/controller-runtime/blob/main/pkg/log/zap/zap.go#L177-L187

Reply:
There is also a flag we inherit from controller-runtime to change this, so I think it's safer to default this to prod https://github.com/kubernetes-sigs/controller-runtime/blob/main/pkg/log/zap/zap.go#L257-L259